app/views/home/index.html.haml

```diff
@@ -12,6 +12,11 @@
             %br
             = link_to "View more details & apply", event_path(@upcoming_event.first), class: "btn btn-primary btn-lg btn-hero"
 
+.container
+  .row
+    .col-md-12
+      = render 'layouts/messages'
```

Comment:
i cant think of anything that we would need to display a message for on a home page (for now). this might be redundant fro before, lets remove the test and remove the message from the layout :)